core/scss/components/main-nav/_main-nav.scss

```diff
@@ -0,0 +1,162 @@
+@charset "UTF-8";
+
+//
+// Main Navigation
+//
+// The primary site navigation that is usually in the header region of a website. It is horizontal on larger devices and collapses on small devices.
+//
+// .su-main-nav--center - Centered main nav
+// .su-main-nav--right - Right-aligned main nav
+//
+// Markup: ../templates/components/main-nav/main-nav.twig
+//
+// Style guide: Components.Main Navigation
+//
+
+.su-main-nav {
+  @include font-smoothing(antialiased);
+  font-family: $font-sans;
+
+  ul {
+    @include margin(0 null);
+    list-style: none;
+  }
+
+  a {
+    text-decoration: none;
+    color: $color-black;
+  }
+}
+
+.su-main-nav__menu-level1 {
+  display: flex;
+  flex-direction: column;
+  background-color: $color-black;
+  width: 100%;
+
+  > li {
+
+    @include margin(null null 0);
+
+    a {
+      @extend %fancyhover;
+      @include padding(.8em .4em .8em 0);
+      display: block;
+      color: $color-white;
+      font-size: .9em;
+      border-bottom: $color-cool-grey solid 1px;
+    }
+  }
+
+  @media #{$breakpoint-lg} {
+    @include padding(null null null 0);
+    flex-direction: row;
+    background-color: transparent;
+
+    > li {
+
+      a {
+        @include padding(null 0 .8em);
+        @include margin(0 .58em);
+        font-size: 1.05em;
+        color: $color-black;
+        font-weight: 600;
+        border-bottom: 0;
+
+        &:hover,
+        &:focus {
+          color: $color-dark-red;
+        }
+
+        &:active {
+          color: $color-black;
+        }
+
+        &:hover {
+          + .su-main-nav__menu-level2 {
+            //display: flex;
+          }
+        }
+      }
+
+      &:first-child {
+        a {
+          @include margin(null null null 0);
+        }
+      }
+
+      &:last-child {
+        a {
+          @include margin(null 0 null null);
+        }
+      }
+
+
+    }
+  }
+}
+
+.su-main-nav__menu-level2 {
+  //display: none;
+  background-color: $color-black;
+  flex-direction: column;
+  position: relative;
+  top: .6em;
+
+  li {
+    a {
+      color: $color-white;
+      font-size: .9em;
+      font-weight: 400;
+
+      &:hover,
+      &:focus {
+        color: $color-white;
+      }
+
+    }
+  }
+}
+
+%fancyhover {
```

Comment:
I am of the opinion that extends, like, mixins, should be in their own file and documented. This is something we haven't encountered yet and have room for discussion and standardization.

Reply:
I agree with you. Perhaps we could talk about this in the Wed meeting.